### DISCUSSION TITLE
Kernel crashing because unknow reason.

### DISCUSSION THREAD

**Discussion by opiepl** [pterodactyl/panel#2874] [Support]
Hi, can somebody check why is my kernel keep crashing after installing wings? 
Link to full kern.log file: https://pastebin.com/DNsAmC19

**Comment by parkervcp** [comment]
You are the only person I have seen with this issue. It looks like your config doesn't like the pterodactyl0 veth port.

**Reply by opiepl** [reply]
It's first time I made a ptero on DC provided ISO (ubuntu 18.04), I think here's the problem

**Comment by gOOvER** [comment]
What Kernel Version?
